graph/typeDefs.graphql

```diff
@@ -884,6 +912,54 @@ input RejectUsernameInput {
   message: String!
 }
 
+# Configurable settings that can be overridden for the Asset.
+input AssetSettingsInput {
+
+  # premodLinksEnable will put all comments that contain links into premod.
+  premodLinksEnable: Boolean
+
+  # moderation is the moderation mode for the asset.
+  moderation: MODERATION_MODE!
+
+  # questionBoxEnable will enable the Question Boxs' content to be visable above
+  # the comment box.
+  questionBoxEnable: Boolean
+
+  # questionBoxContent is the content of the Question Box.
+  questionBoxContent: String
+
+  # questionBoxIcon is the icon for the Question Box.
+  questionBoxIcon: String
+}
+
+# UpdateAssetStatusInput contains the input to change the status of a comment as
+# it relates to being open/closed for commenting.
+input UpdateAssetStatusInput {
```

Comment:
See above

Reply:
I've updated the comment to reflect the behaviour. You should update all fields when posting, you cannot specify a subset. This is because for some fields, the `null` value is used.